From: Esme Povirk <esme@codeweavers.com>
Subject: [PATCH 4/5] mscoree: Update Wine Mono to 7.1.1.
Message-Id: <20220128190054.662812-4-esme@codeweavers.com>
Date: Fri, 28 Jan 2022 13:00:53 -0600
In-Reply-To: <20220128190054.662812-1-esme@codeweavers.com>
References: <20220128190054.662812-1-esme@codeweavers.com>

Signed-off-by: Esme Povirk <esme@codeweavers.com>
---
This version installs and registers diasymreader.dll, so we need to at
least have enough of that dll to register it first.

 dlls/appwiz.cpl/addons.c       | 4 ++--
 dlls/mscoree/mscoree_private.h | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/dlls/appwiz.cpl/addons.c b/dlls/appwiz.cpl/addons.c
index b4da6137d72..71919f225c8 100644
--- a/dlls/appwiz.cpl/addons.c
+++ b/dlls/appwiz.cpl/addons.c
@@ -58,10 +58,10 @@ WINE_DEFAULT_DEBUG_CHANNEL(appwizcpl);
 #define GECKO_SHA "???"
 #endif
 
-#define MONO_VERSION "7.0.0"
+#define MONO_VERSION "7.1.1"
 #if defined(__i386__) || defined(__x86_64__)
 #define MONO_ARCH "x86"
-#define MONO_SHA "b37e6fc9e590e582243dc25d72a5fcc330c3a7970dfdc98a7a81d23845ba8900"
+#define MONO_SHA "9dc8e5603b7bc64354eb94ae4ea0f6821424767a3ff44ff0d19e346a490c11ea"
 #else
 #define MONO_ARCH ""
 #define MONO_SHA "???"
diff --git a/dlls/mscoree/mscoree_private.h b/dlls/mscoree/mscoree_private.h
index 1ef5ac200f3..2e72ea5370b 100644
--- a/dlls/mscoree/mscoree_private.h
+++ b/dlls/mscoree/mscoree_private.h
@@ -45,7 +45,7 @@ extern HRESULT assembly_get_runtime_version(ASSEMBLY *assembly, LPSTR *version)
 extern HRESULT assembly_get_vtable_fixups(ASSEMBLY *assembly, VTableFixup **fixups, DWORD *count) DECLSPEC_HIDDEN;
 extern HRESULT assembly_get_native_entrypoint(ASSEMBLY *assembly, NativeEntryPointFunc *func) DECLSPEC_HIDDEN;
 
-#define WINE_MONO_VERSION "7.0.0"
+#define WINE_MONO_VERSION "7.1.1"
 
 /* Mono embedding */
 typedef struct _MonoDomain MonoDomain;

-- 
2.35.1

